js/battle.js

```diff
@@ -4761,6 +4761,7 @@ var Battle = (function () {
 					actions += "" + poke.getName() + " fell straight down!";
 					poke.removeVolatile('magnetrise');
 					poke.removeVolatile('telekinesis');
+					if (poke.lastmove === 'fly' || poke.lastmove === 'bounce') poke.sprite.animReset();
```

Comment:
"The lastmove check is there in case a levitating mon gets hit by No Guard Smack Down while using Dig or Dive.", as in that case the Dig/Dive anim would also be canceled.